src/pymor/vectorarrays/interface.py

```diff
@@ -407,7 +407,7 @@ def pairwise_inner(self, other, product=None):
 
     @Deprecated(pairwise_inner)
     def pairwise_dot(self, other):
-        return self.pariwise_inner(other)
+        return self.pairwise_inner(other)
```

Comment:
Seems this was never exercised in tests before, is `pairwise_inner` now?

Reply:
No. Maybe it was not so wise to remove the test for `pairwise_dot`. Was too lazy to open a separate PR for this.